### PR TITLE
Feature/18 popular people item click listener

### PR DESCRIPTION
**Pull request by MuhammadAbuBakar9539**

### CODE REVIEW

File: app/src/test/java/com/example/themoviedb/PopularPeopleViewModelTest.kt [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+        //Then
+        Assert.assertEquals(model, viewModelTest.getPopularPeople().value)
+        verify(popularPeopleObserver).onChanged(model)
+    }
+
+    @Test
+    fun `fetch Popular People call UnSuccessful`() {
+        //Given
+        val error = "error"
+        `when`(repository.getPopularPeople()).thenReturn(Single.error(NullPointerException(error)))
```
Comment (EMHankombo) [inline app/src/test/java/com/example/themoviedb/PopularPeopleViewModelTest.kt]: Would be better to change this to a more generic RuntimeException(error) than NullPointerException

File: app/src/main/res/layout/activity_person_detail.xml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
```
Comment (EMHankombo) [inline app/src/main/res/layout/activity_person_detail.xml:2]: Is this doing anything?

**Review by EMHankombo — CHANGES_REQUESTED** [review]
Address minor change

File: app/src/main/res/layout/activity_person_detail.xml (hunk 2 above)
Comment (MuhammadAbuBakar9539) [reply app/src/main/res/layout/activity_person_detail.xml:2]: its just a fake activity in order to run intent

**Review by EMHankombo — APPROVED** [review]

**Pull request merged**